src/api/aptosConfig.ts

```diff
@@ -154,6 +168,10 @@ export class AptosConfig {
     this.fullnodeConfig = settings?.fullnodeConfig ?? {};
     this.indexerConfig = settings?.indexerConfig ?? {};
     this.faucetConfig = settings?.faucetConfig ?? {};
+    this.transactionGenerationConfig = settings?.transactionGenerationConfig ?? {
```

Comment:
what is the reasoning in adding it in the global config vs adding it in the transaction option prop? https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/transactions/types.ts#L116

Reply:
Because it has to set on every call, which is annoying. Theres a default, but it's not configurable.  This makes the defaul val configurable, while allowing for override